scanner/config/config.go

```diff
@@ -18,13 +19,20 @@ var (
 		HTTPListenAddr: "127.0.0.1:9443",
 		GRPCListenAddr: "127.0.0.1:8443",
 		Indexer: IndexerConfig{
-			Enable:          true,
-			DBConnString:    "postgresql:///postgres?host=/var/run/postgresql",
+			Enable: true,
+			Database: Database{
+				ConnString:   "host=/var/run/postgresql",
+				PasswordFile: "",
```

Comment:
This will be set to `/run/secrets/stackrox.io/secrets/password` in the deployment config

Reply:
?

Reply:
I guess you were asking a question, see https://github.com/stackrox/stackrox/pull/7939 to understand how this will be used.